### PR TITLE
Add mediatek support

### PR DESCRIPTION
**Pull request by plbossart**

### CODE REVIEW

File: arm64-nobloat-defconfig [hunk 1]
```diff
@@ -77,3 +76,5 @@
 # CONFIG_MEDIA_SUPPORT is not set
 # CONFIG_RC_CORE is not set
 CONFIG_IMX_DSP=y
+CONFIG_PINCTRL_MT8195=y
```
Comment (yaochunhung) [inline arm64-nobloat-defconfig:79]: No sure if it is necessary since I don't set it in my local defconfig.

Comment (yaochunhung) [reply arm64-nobloat-defconfig:79]: @plbossart  Thanks. if also need to consider sound soc part (machine driver). please also add CONFIG_SND_SOC_MT8195 and  CONFIG_SND_SOC_MT8195_MT6359_RT1019_RT5682. below configs are that I use currently. thanks.  
 CONFIG_SND_SOC_MT8195=y
CONFIG_SND_SOC_MT8195_MT6359_RT1019_RT5682=y
CONFIG_SND_SOC_SOF_OF=y
CONFIG_SND_SOC_SOF_TOPLEVEL=y
CONFIG_SND_SOC_SOF_MTK_TOPLEVEL=y
CONFIG_SND_SOC_SOF_MT8195=y

Comment (plbossart) [reply arm64-nobloat-defconfig]: done!

Comment (yaochunhung) [reply arm64-nobloat-defconfig]: thanks.

**Review by yaochunhung — APPROVED** [review]

**Comment by plbossart** [comment]
ok let's leave @dbaluta a chance to comment.

**Comment by dbaluta** [comment]
@plbossart @plbossart Looks good to me! Thanks!

**Pull request merged**